Makefile

```diff
@@ -5,8 +5,13 @@ obj-m += $(MODULE_NAME).o
 KERNEL_HEAD := $(shell uname -r)
 KERNELDIR := /lib/modules/$(KERNEL_HEAD)/build
 PWD := $(shell pwd)
+ARCH := $(shell uname -p)
```

Comment:
`uname -p` should be `uname -m`, right?

Reply:
yes I should use uname -m instead of uname -p